README.md

```diff
@@ -18,6 +18,7 @@ following links:
 - [Shelley ledger formal specification](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/shelleyLedgerSpec/latest/download-by-type/doc-pdf/ledger-spec): the formal mathematical specification of the Shelley era ledger rules.
 - [Shelley binary format specification (CDDL)](https://github.com/input-output-hk/cardano-ledger-specs/tree/master/shelley/chain-and-ledger/shelley-spec-ledger-test/cddl-files): the binary formats for the Shelley ledger using CBOR CDDL schema notation.
 - [Non-integer calculations specification](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/nonIntegerCalculations/latest/download-by-type/doc-pdf/non-integer-calculations): details on the parts of the Shelley specification that use real numbers.
+- [Stake pool ranking specification](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs/specs.pool-ranking/latest/download-by-type/doc-pdf/pool-ranking): details for a robust stake pool ranking mechanism.
```

Comment:
This link does not work right now, but I expect it to work as soon as there is a a first build in master.  (a [link from this PR branch](https://hydra.iohk.io/job/Cardano/cardano-ledger-specs-pr-1816/specs.pool-ranking/latest/download-by-type/doc-pdf/pool-ranking) does work)